velox/functions/lib/ArraySort.cpp

```diff
@@ -495,9 +505,36 @@ core::CallTypedExprPtr asArraySortCall(
 
 } // namespace
 
+std::shared_ptr<exec::VectorFunction> makeArraySortLambdaFunction(
+    const std::string& /*name*/,
+    const std::vector<exec::VectorFunctionArg>& inputArgs,
+    const core::QueryConfig& /*config*/,
+    bool ascending,
+    bool throwOnNestedNull) {
+  VELOX_CHECK_EQ(inputArgs.size(), 2);
```

Comment:
nit: use VELOX_USER_CHECK_EQ instead as it does not indicate a bug but a user error.